cmd/containerd-shim-runhcs-v1/task_wcow_podsandbox.go

```diff
@@ -236,6 +237,25 @@ func (wpst *wcowPodSandboxTask) DumpGuestStacks(ctx context.Context) string {
 	return ""
 }
 
+func (wpst *wcowPodSandboxTask) Update(ctx context.Context, req *task.UpdateTaskRequest) error {
+	resources, err := typeurl.UnmarshalAny(req.Resources)
```

Comment:
Can we just return up here if we're not isolated and skip the unmarshalling/verifying?

Reply:
That scenario really won't ever happen unless we repurpose the wcowpodsandboxtask for something else, but I don't think we will. I added a check more as a sanity check and to make sure we don't panic if something goes REALLY wrong.

Reply:
We repurposed it for job containers is why :)

Reply:
haha should've led with that. We re-use this for them as we don't launch a pause container also

Reply:
😥okay I'll fix that then lol